### DISCUSSION TITLE
Go to watch face on wake up

### DISCUSSION THREAD

**Discussion by IAmAMaggot** [InfiniTimeOrg/InfiniTime#1826] [Ideas]
Hi!
Great fan here! I have a problem, that I often accidentally swipe over the screen after lifting my arm. I would love to see an option, that on every screen wake up, the watch face is shown. This should be a simple thing wouldn't it? I hate it when I want to see the time and lifting my arm only leads to "No notifications to display".

**Comment by vkareh** [comment]
I've created a PR for this: https://github.com/InfiniTimeOrg/InfiniTime/pull/1980

**Comment by SteveAmor** [comment]
This feature request is now in V1.15. Can this discussion be closed?

**Reply by FintasticMan** [reply]
Yes, it can!
